README.md

```diff
@@ -37,6 +37,18 @@ make dev
 
 will start a local development instance in Docker.
 
+### Testing Changes
+
+Run the full test suite:
+
+```
+make test
+```
+
+Due to Codeship limitations, it's not possible to run the full test suite in Codeship because you cannot disable the network adapter.
```

Comment:
What is tested via codeship vs local running of the suite?

Reply:
The tests cover fetching data to fill the cache, serving cached content, and fetching new content when the cache expires. There is also test that cached content is served when there is no network. That part doesn't run in codeship.